cmd/bootstrap/template.go

```diff
@@ -9,7 +9,10 @@ serviceAccount:
 env:
   open:
     ALLOW_OVERWRITE: true
-    DISABLE_API: false`
+    DISABLE_API: false
+probes:
+  readiness:
+    initialDelaySeconds: 10`
```

Comment:
The default is 5 secs. Increasing it to try and make posting the tarball after first install more reliable.